tests/link/pytorch/test_basic.py

```diff
@@ -335,7 +335,7 @@ def test_pytorch_OpFromGraph():
     ofg_2 = OpFromGraph([x, y], [x * y, x - y])
 
     o1, o2 = ofg_2(y, z)
-    out = ofg_1(x, o1) + o2
+    out = ofg_1(x, o1) / o2
```

Comment:
This probably could be it's own commit, but c'est la vie.

Reply:
Torch makes an individual guard for each `Elemwise._check_runtime_shapes`. So we would have a guard for multiplication, subtraction, addition and division. I just wanted to keep all of them unique so I could let stuff blow up appropriately.